pkg/controller/constrainttemplate/stats_reporter.go

```diff
@@ -0,0 +1,158 @@
+package constrainttemplate
+
+import (
+	"context"
+	"time"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/metrics"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	ctCount        = "constraint_templates"
```

Comment:
update `ctCount` and `ctCountDesc` var names to match also

Reply:
match in what way?

Reply:
meant similar naming convention, so would be called `constraintTemplatesName` etc

Reply:
I have the same issue with constraintTemplatesName as I do with `constraintsName`, that it may collide with other, more expected uses of that variable name. Is there a more semantic name we could use?

Additionally, tying the name of the variable to the contents of the string it stores somewhat defeats the purpose of abstracting the string, so whatever semantic name we choose should be insensitive to the exact string value, IMO.

Reply:
Closing this as dupe of above discussion.